comp/core/workloadmeta/types.go

```diff
@@ -340,6 +340,8 @@ type ContainerPort struct {
 	Name     string
 	Port     int
 	Protocol string
+	HostPort uint16
+	HostIP   string
```

Comment:
❓ 

I understand adding `HostPort` as the container port is usually mapped to a host port. But could you please explain how `HostIP` fits in this type definition?

Reply:
Here it's 1:1 mapping with https://github.com/DataDog/datadog-agent/blob/7d3819e712f3743a7fc3617c6ff3c9a8be56fe1a/pkg/util/ecs/metadata/v3or4/types.go#L71-L76